### ISSUE TITLE
How to add new plants/relationships?

### ISSUE THREAD

**Issue by coatesap** [serlo/PlantBuddies#23]
Hi, this is a great tool, and I'd love to contribute some additional plants/relationships. 

Would it be possible to add a "Contributing" section to the readme that explains the project setup and the basic steps to add new plant and relationship data (with any gotchas/things to consider)?

If not, that's fine - it looks reasonably straight-forward. The only immediate question that jumped out at me was that `data/relations-data.js` looked like it had been minified, but I couldn't see a non-minified version to make edits to?

**Comment by elbotho** [comment]
Hey there, thanks for reaching out!
There was an "backend" at the beginning which we used to get started with the bulk of the data.

Since then (hacky backend aside) I put quite some thought into a proper way to manage the data for this project.
But to be honest there was not a lot of interest in collaboration so far so I never finished anything.

So my questions:
- Did you plan on adding a lot of relations/plants? What are your sources?
- Would you be interested working on setting up a simple, stable and usable process?

**Comment by coatesap** [comment]
No, I wasn't planning on adding a lot of new plants - just some of the more common (UK) garden vegetables like Courgette (Zucchini). 

I'm not aware of 100% authoritative sources for companion planting, but would attempt to find 2-3 relatively reputable sources for any new relationships. Sites like https://www.farmersalmanac.com/companion-planting-guide-31301 that provide an explanation for any recommendations, for instance.

I don't have a lot of available time, but would be willing to put in a few hours here and there.

**Comment by elbotho** [comment]
**Quickfix**:
Maybe we should just rename this one  😉 
https://plantbuddies.serlo.org/🌿/marrow

**Comment by coatesap** [comment]
Ha - I didn't even think of searching for "Marrow". I guess they're not grown as Marrows that often in the UK (or the US, AFAIK) so renaming to Courgette or Zucchini would definitely be the quick fix!

A 'nice to have' would be if the autocomplete could also work on the alternative names.
